EpiAware/src/utils/dist.jl

```diff
@@ -167,3 +165,17 @@ function NegativeBinomialMeanClust(μ, α)
     r = μ^2 / ex_σ²
     return NegativeBinomial(r, p)
 end
+
+"""
+Create a half-normal prior distribution with the specified mean.
+
+# Arguments
+- `prior_mean::AbstractFloat`: The mean of the prior distribution.
+
+# Returns
+- `Truncated{Normal}`: The half-normal prior distribution.
+
+"""
+function _make_halfnormal_prior(prior_mean::AbstractFloat)
+    return truncated(Normal(0.0, prior_mean * sqrt(pi) / sqrt(2)), 0.0, Inf)::Distribution
```

Comment:
For interest adding the type (distribution) is optional here right? What informa your choice here? Clarity or an expectation of a performance difference?

Reply:
This is more a quick fingered typo.

Reply:
Removed